Doc/whatsnew/3.9.rst

```diff
@@ -827,6 +827,11 @@ Changes in the Python API
   in the object itself.
   (Contributed by Serhiy Storchaka in :issue:`40257`.)
 
+* :meth:`asyncio.loop.shutdown_default_executor` has been added added to
```

Comment:
```suggestion
* :meth:`asyncio.loop.shutdown_default_executor` has been added to
```

Reply:
Oops, I have no idea how that happened. Thanks for mentioning it. :-)